### ISSUE TITLE
Error:  Can't load '.../blib/arch/auto/PDL/SVDSLEPc/SVDSLEPc.so' for module PDL::SVDSLEPc

### ISSUE THREAD

**Issue by mohawk2** [moocow-the-bovine/PDL-SVDSLEPc#1]
We can't currently build this distro in our downstream testing (as created by @zmughal), this (https://github.com/PDLPorters/devops/blob/master/data/project.yml#L129) shows:

```yaml
      skip: |
        Error:  Can't load '.../blib/arch/auto/PDL/SVDSLEPc/SVDSLEPc.so' for module PDL::SVDSLEPc:
        Fix is:
          export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libpetsc_real.so.3.15.5:/usr/lib/x86_64-linux-gnu/libslepc_real.so.3.15.2 ; make test
```

**Comment by moocow-the-bovine** [comment]
That's very odd.  The EU::MM Makefile should be getting e.g. `-lslepc_real` in its `LDFLAGS` and `LDDLFLAGS` via `$(SLEPC_LIB)` as included from `$(SLEPC_DIR)/lib/slepc/conf/slepc_variables`.

That said, I'm not very attached to this package: it's never been in production use, and was pretty quick & dirty to begin with.  A "real" PETSc/SLEPc wrapper would be preferable, but beyond my ability to write, test & maintain.  Perhaps it's best if we let this module rot until such time as someone actually needs it.

**Comment by mohawk2** [comment]
Having had a quick go, and seen:
- the vast number of Ubuntu packages needed (which intriguingly included OpenCL stuff)
- `mpicc` appears to put all the `-l` flags before our generated `.o` (which will be why it's not linking in the symbols), for a reason that is not immediately visible (I think there's a whole world of MPI stuff opening up there, of which I am for now ignorant)

I think that may be for the best for now.

**Comment by moocow-the-bovine** [comment]
I agree the openMPI and openCL stuff looks potentially very useful, but it's a lot to digest, and I never really managed it.  I'll leave this issue open (presumably ad infinitum) for the record.
